apps/test/util/netsimTestUtils.js

```diff
@@ -130,6 +133,22 @@ var fakeStorageTable = function () {
     },
 
     /**
+     * @param {!Object[]} values
+     * @param {!NodeStyleCallback} callback
+     */
+    multiCreate: function (values, callback) {
+      values.forEach(function (value) {
+        log_ += 'create[' + JSON.stringify(value) + ']';
+
+        value.id = rowIndex_;
+        rowIndex_++;
+        tableData_.push(value);
+      });
+
+      callback(null, values);
```

Comment:
Was going to question whether the `values` passed to the callback had the IDs added to it, but it does.  Nicely done.